File: Class_Member/2.py
class Cs:
    @staticmethod
    def static_method():
        print("Static method")

    @classmethod
    def class_method(cls):
        print("Class method")

    def instance_method(self):
        print("Instance method")
i = Cs()
Cs.static_method() # Static method
Cs.class_method() # Class method
i.instance_method() # Instance method

File: Class_Member/2.rb
class Cs
  def Cs.class_method()
    p "Class method"
  end
  def instance_method()
    p "Instance method"
  end
end
i = Cs.new()
Cs.class_method() # Class method
i.instance_method() # Instance method
#Cs.instance_method() 오류발생
#i.class_method() 오류발생
